diff -up ./Makefile.fix-build ./Makefile
--- ./Makefile.fix-build	2022-02-14 14:22:51.602012446 -0500
+++ ./Makefile	2022-02-14 14:23:26.520527935 -0500
@@ -6,9 +6,9 @@ SHELL = /bin/bash
 # version is moved to a separate repo and release process.
 export IMAGE_VERSION = v1.17.0
 # Build-time variables to inject into binaries
-export SIMPLE_VERSION = $(shell (test "$(shell git describe)" = "$(shell git describe --abbrev=0)" && echo $(shell git describe)) || echo $(shell git describe --abbrev=0)+git)
-export GIT_VERSION = $(shell git describe --dirty --tags --always)
-export GIT_COMMIT = $(shell git rev-parse HEAD)
+export SIMPLE_VERSION = v1.17.0
+export GIT_VERSION = $(SIMPLE_VERSION)
+export GIT_COMMIT = ""
 export K8S_VERSION = 1.21
 # TODO: bump this to 1.21, after kubectl `--generator` flag is removed from e2e tests.
 export ENVTEST_K8S_VERSION = 1.21.1
